app/models/piece.rb

```diff
@@ -93,4 +93,20 @@ def remains_on_board?(to_x:, to_y:)
       false
     end
   end
+
+  def successful_capture(to_x:, to_y:)
+    return false if obstructed_diagonally?(to_x:, to_y:)
+    return false if obstructed_vertically?(to_x:, to_y:)
+    return false if obstructed_horizontally?(to_x:, to_y:)
+    current_color = current_piece.color
+    target_piece = Piece.where(x_position: to_x, y_position: to_y, active: true, color: !current_color)  # Add requirement that target MUST be opponent's color
+    if target_piece.exists?
+      return false if target_piece == king  # King can't be captured, only checkmated
```

Comment:
It dawned on me, I think we can delete this line (#104) because:
1) When player A puts B's king in check, it is not a capture **and**
2) On the game's next move, player B **must** get his king out of check.
3) Thus, when it's A's turn again, the "B" king will never be a possible capture.